src/Reveal.Sdk.Dom/Visualizations/DoughnutChartVisualization.cs

```diff
@@ -3,10 +3,24 @@
 
 namespace Reveal.Sdk.Dom.Visualizations
 {
+    /// <summary>
+    /// The doughnut visualization is similar to a pie chart, proportionally illustrating the occurrences of a variable.

```

Comment:
no need to compare to the pie chart. Just tell them what the doughnut chart does